services/net/image/appsettings.json

```diff
@@ -15,7 +15,11 @@
     "OutputPath": "/data/image",
     "Command": "ffmpeg",
     "MediaTypes": "Image",
-    "PrivateKeysPath": "keys"
+    "PrivateKeysPath": "keys",
+    "PrivateKeyFileName": "id_rsa",
+    "InputPath": "/dsk98/binaryroot/",
+    "HostName": "scharnhorst.tno.gov.bc.ca",
```

Comment:
I wouldn't include the `HostName` or `UserName` in the default config values.  They're only for local testing.